### DISCUSSION TITLE
Re-introduced bug in Nemo 5.8.5

### DISCUSSION THREAD

**Discussion by jemehlbox** [linuxmint/discussions#171] [Questions]
Nemo in Mint 20 - 20.3 Cinamon (for my installation Nemo 5.2.4) has a bug. When you double click on "Network" and try to drill down to your SMB share, you get an error.
This bug is fixed in Mint 21.1 and Mint 21.2 with Nemon 5.8.4. You can double click on "Network" and drill down to your share to get it mounted.
When I updated Nemo to 5.8.5, that bug was present again. No mounting, instead the error message.
I timeshifted back (thanks that this is available) to Nemo 5.8.4, and it worked again.
Has anybody else experienced this?

Thanks
jemehlbox

**Comment by mtwebster** [comment]
Can you be a bit more detailed about this?

what exactly is the error? (screenshot if you can)

Does this happen always or can you mount it sometimes and sometimes not?

**Comment by jemehlbox** [comment]
I would, but the error went away.
An hour ago I updated Firefox and in error also Nemo on my Mint 21.2 laptop. The error appeared. I timeshifted again, and the error went away. (Nemo 5.8.4 again). To get you a screenshot I updated Nemo to 5.8.5 again and the error did NOT reappear.
But to make things worse for me, I doublechecked with my Mint 20.3 laptop (Nemo 5.2.4): NO error.
I have no explanation...

**Comment by jemehlbox** [comment]
It now works in my 20.3, 21.1, and 21.2 installation. There is no use in spending more time on it. Therefore I close this.
